src/pkg/cli/tail.go

```diff
@@ -362,7 +362,7 @@ func streamLogs(ctx context.Context, provider client.Provider, projectName strin
 					cancel() // TODO: stuck on defer Close() if we don't do this
 					return nil
 				}
-				return nil
+				continue
```

Comment:
@jordanstephens this was introduced in #1169, I am not sure changing it back to `return nil` would have any impact on the price estimate feature, please take a look, thx.